### PR TITLE
Feature/tt 1172 history text

### PR DESCRIPTION
**Pull request by gtryus**

### CODE REVIEW

File: src/components/Transcriber.tsx [hunk 1]
```diff
@@ -493,6 +489,32 @@ export function Transcriber(props: IProps) {
...
+
+  const sectionDescriptionStyle = {
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+  };
+
+  const historyStyle = { overflow: 'auto', height: boxHeight };
+
+  const commentStyle = { overflow: 'auto' };
+
```
Comment (sarahentzel) [inline src/components/Transcriber.tsx]: should this use makeStyles to be consistent with other screens?

File: src/components/Transcriber.tsx [hunk 2]
```diff
@@ -549,36 +571,25 @@ export function Transcriber(props: IProps) {
...
-      >
+      <Paper className={classes.paper} onKeyDown={handleKey} style={paperStyle}>
         <Grid container direction="column">
           <Grid container direction="row" className={classes.row}>
-            <Grid
-              item
-              xs={9}
-              style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}
-            >
+            <Grid item xs={9} style={sectionDescriptionStyle}>
```
Comment (sarahentzel) [inline src/components/Transcriber.tsx]: do you think the {9} needs to be in a constant?

File: src/components/Transcriber.tsx (hunk 1 above)
Comment (gtryus) [reply src/components/Transcriber.tsx]: I moved all but the dynamic calculations to makeStyles. I think the dynamic styles (like the height or width based on resizing may need to be done this way or they won't update properly.

File: src/components/Transcriber.tsx (hunk 2 above)
Comment (gtryus) [reply src/components/Transcriber.tsx]: For me having 9 here might be clearer than a constant if I don't know the value or have to go look up the value.

**Pull request merged**